part-iv-advanced-tuning/distribution-considerations.md

```diff
@@ -0,0 +1,27 @@
+# Distribution Considerations
+
+One of the core properties of applications built using Axon is Location Transparency - being able to run application on several nodes increases scalability, responsiveness and resilience. This section summarizes things that should be taken into consideration while distributing Axon-based applications.
+
+## Distributing Commands
+
+More details on how Distributing the Command Bus works can be found [here](/part-iii-infrastructure-components/command-dispatching.md#distributing-the-command-bus). Two the most important components in a `DistributedCommandBus` are `CommandBusConnector` and `CommandRouter`. 
```

Comment:
I don't think this is entirely correct when it comes to AxonHub. When using AxonHub, you're not using the DistributedCommandBus in combination with an AxonHub CommandBusConnector. Instead, you're using an AxonHubCommandBus which directly implements to CommandBus interface. It takes a RoutingStrategy instead of a Command Router.

I would roughly write it up like this: for distributing commands, there's the option to use the AxonHubCommandBus, and the option of using the DistributedCommandBus which requires you to choose and set up a CommandBusConnector.